pkg/square/builder_test.go

```diff
@@ -2,253 +2,152 @@ package square_test
+// 		{"one hundred large pfb medium block", 100, 100, 100000, appconsts.DefaultGovMaxSquareSize},
+// 		{"mixed transactions large block", 100, 100, 100000, appconsts.DefaultGovMaxSquareSize},
+// 		{"mixed transactions large block 2", 1000, 1000, 10000, appconsts.DefaultGovMaxSquareSize},
+// 		{"mostly transactions large block", 10000, 1000, 100, appconsts.DefaultGovMaxSquareSize},
+// 		{"only small pfb large block", 0, 10000, 1, appconsts.DefaultGovMaxSquareSize},
+// 	}
+// 	for _, tt := range tests {
+// 		t.Run(tt.name, func(t *testing.T) {
+// 			rand := tmrand.NewRand()
+// 			signer, err := testnode.NewOfflineSigner()
+// 			require.NoError(t, err)
+// 			txs := generateMixedTxs(signer, rand, tt.normalTxs, tt.pfbCount, 1, tt.pfbSize)
+// 			square, _, err := square.Build(txs, appconsts.LatestVersion, appconsts.DefaultGovMaxSquareSize)
+// 			require.NoError(t, err)
+// 			require.EqualValues(t, tt.expectedSquareSize, square.Size())
+// 		})
+// 	}
+// }
 
 func TestBuilderRejectsTransactions(t *testing.T) {
-	builder, err := square.NewBuilder(2, appconsts.LatestVersion) // 2 x 2 square
+	builder, err := square.NewBuilder(2, 64) // 2 x 2 square
 	require.NoError(t, err)
 	require.False(t, builder.AppendTx(newTx(shares.AvailableBytesFromCompactShares(4)+1)))
 	require.True(t, builder.AppendTx(newTx(shares.AvailableBytesFromCompactShares(4))))
 	require.False(t, builder.AppendTx(newTx(1)))
 }
 
-func TestBuilderRejectsBlobTransactions(t *testing.T) {
-	ns1 := ns.MustNewV0(bytes.Repeat([]byte{1}, ns.NamespaceVersionZeroIDSize))
-	testCases := []struct {
-		blobSize []int
-		added    bool
-	}{
-		{
-			blobSize: []int{shares.AvailableBytesFromSparseShares(3) + 1},
-			added:    false,
-		},
-		{
-			blobSize: []int{shares.AvailableBytesFromSparseShares(3)},
-			added:    true,
-		},
-		{
-			blobSize: []int{shares.AvailableBytesFromSparseShares(2) + 1, shares.AvailableBytesFromSparseShares(1)},
-			added:    false,
-		},
-		{
-			blobSize: []int{shares.AvailableBytesFromSparseShares(1), shares.AvailableBytesFromSparseShares(1)},
-			added:    true,
-		},
-		{
-			// fun fact: three blobs increases the size of the PFB to two shares, hence this fails
-			blobSize: []int{
-				shares.AvailableBytesFromSparseShares(1),
-				shares.AvailableBytesFromSparseShares(1),
-				shares.AvailableBytesFromSparseShares(1),
-			},
-			added: false,
-		},
-	}
-
-	for idx, tc := range testCases {
-		t.Run(fmt.Sprintf("case%d", idx), func(t *testing.T) {
-			builder, err := square.NewBuilder(2, appconsts.LatestVersion)
-			require.NoError(t, err)
-			txs := generateBlobTxsWithNamespaces(t, ns1.Repeat(len(tc.blobSize)), [][]int{tc.blobSize})
-			require.Len(t, txs, 1)
-			blobTx, isBlobTx := blob.UnmarshalBlobTx(txs[0])
-			require.True(t, isBlobTx)
-			require.Equal(t, tc.added, builder.AppendBlobTx(blobTx))
-		})
-	}
-}
+// func TestBuilderRejectsBlobTransactions(t *testing.T) {
```

Comment:
[blocking] is this intended to be committed commented out? Instead of merging commented code can we:
1. delete if no longer relevant
1. uncomment and fix 
2. worst case: delete with a GH issue to re-add?